herepy/routing_api.py

```diff
@@ -66,14 +66,15 @@ def __get(
             if json_data.get(key) is not None:
                 return response_cls.new_from_jsondict(json_data)
             else:
-                raise error_from_routing_service_error(json_data)
-        else:
-            raise HEREError(
```

Comment:
`InvalidRequestError`s from v7 calls would not get caught here and instead the `HereError` is thrown and the error message isn't helpful since it is thought of an unhandled.